Common/Data/UniverseSelection/BenchmarkUniverse.cs

```diff
@@ -0,0 +1,107 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Linq;
+using QuantConnect.Interfaces;
+using QuantConnect.Securities;
+
+namespace QuantConnect.Data.UniverseSelection
+{
+    /// <summary>
+    /// Represents the universe containing only the selected benchmark security.
+    /// This universe keeps the benchmark security and its subscription separate
+    /// from the algorithm securities. Universe selection will fire only once,
+    /// when the benchmark security is added to the universe.
+    /// </summary>
+    public class BenchmarkUniverse : Universe, INotifyCollectionChanged
+    {
+        /// <summary>
+        /// Event fired when a symbol is added or removed from this universe
+        /// </summary>
+        public event NotifyCollectionChangedEventHandler CollectionChanged;
+
+        /// <summary>
+        /// Gets the settings used for subscriptions added for this universe (unused in this universe)
+        /// </summary>
+        public override UniverseSettings UniverseSettings => null;
```

Comment:
Perhaps a comment why we're setting this to null -- and does this cause potential null ref exceptions elsewhere in the code -- and if not, maybe a little doc on *why* this is safe.

Reply:
Good point!
The only potential null refs I can see are these two:
- in `Universe.CanRemoveMember`: should never be called, but solved by adding the override in `BenchmarkUniverse` 
- in `BacktestingSetupHandler.GetMaximumRuntime`

Thinking about it, might be safer to create an instance with default values (even if they're not actually used) ?